### ISSUE TITLE
improve resolving path (for webpack alias)

### ISSUE THREAD

**Issue by anorudes** [AsaAyers/js-hyperclick#25]
Hello, nice extenstion, but i'am have question - can you improve resolving path?
For sublime text we have this plugin: https://github.com/titoBouzout/Open-Include
it works with webpack alias (for example: import { test } from 'utils/test.js' } 

```
Resolving:

Will try to resolve to:

Absolute path to URL (e.g. https?://...)
Relative to current view
Relative to current view minus 1 folder
Relative to current view minus 2 folders
Relative to project folders
Relative to project folders minus 1 folder
Relative to project folders minus 2 folders
Absolute path
If everything else fails will also look into:

Relative to the folder of all opened views
Relative to all sub-folders
Relative to all parent folders
```

**Comment by AsaAyers** [comment]
I'm specifically following Node's resolution order. There is an open issue for supporting [other module directories](https://github.com/AsaAyers/js-hyperclick/issues/17), but I haven't found a good solution that works per-project yet.

If someone is able to find a solution for that it seems like it should solve your issue here.

**Comment by AsaAyers** [comment]
Resolved in https://github.com/AsaAyers/js-hyperclick/pull/32

The readme has the details on how you can configure your `package.json` for this to work.

**Comment by AsaAyers** [comment]
I just published v1.15 with a new feature I'm calling "Custom Resolvers". It will allow you to add a JS file to your project that can implement your aliases. [PR and documentation](https://github.com/AsaAyers/js-hyperclick/pull/105)
